careerfairsystems/arkad-app
# Upgrade master to Android 9 (Api 28)

Changed targetSdk and buildToolsVersion in android/build.gradle to 28. bundled images and resources as they would otherwise not be included in build. edited gradle.properties to supress a warning about apt and update a deprecated command (compile / implementation). This seems to break hot-reload, see https://github.com/facebook/react-native/issues/18899. For future bundling needs, the following commands might need to be used

react-native bundle --platform android --dev false --entry-file index.js \ --bundle-output android/app/src/main/assets/index.android.bundle \ --assets-dest android/app/src/main/res/

react-native bundle --platform android --dev false --entry-file index.js --bundle-output android/app/src/main/assets/index.android.bundle --assets-dest android/app/src/main/res

(The commands are almost identical, but it still seems to bundle different things)

## Change description

Upgrades to Android 9. 

## How to verify

Start the app and make sure all functionality is still working. 
-Known issue: The combination of Api 28 and React-Native seems to break hot reload, but there is a dirty workaround here : https://github.com/facebook/react-native/issues/18899. It needs to be performed locally on the computer and hence can't be added to the project.

## Issues fixed

#205
